allennlp/data/dataset_readers/penn_tree_bank.py

```diff
@@ -105,7 +107,7 @@ def text_to_instance(self, # type: ignore
         fields: Dict[str, Field] = {"tokens": text_field}
 
         if self._use_pos_tags and pos_tags is not None:
-            pos_tag_field = SequenceLabelField(pos_tags, text_field, "pos_tags")
+            pos_tag_field = TextField([Token(x) for x in pos_tags], token_indexers=self._pos_tag_indexers)
```

Comment:
Do you really want this to be a `TextField`?  Seems simpler to just keep this as a `SequenceLabelField` and instantiate an `Embedding` with the right number of entries from the label vocabulary.

Reply:
Ugh I actually did it like that originally.

Reply:
Why did you switch?  Which one is better?